src/odemis/acq/move.py

```diff
@@ -81,6 +89,48 @@ def getCurrentPositionLabel(current_pos, stage):
     if coating_progress is not None:
         return LOADING_PATH
 
+    # Check the current position is near the line between DEACTIVE and COATING
+    alignment_path = getMovementProgress(current_pos, stage_deactive, stage_alignment)
+    if alignment_path is not None:
+        return LOADING_PATH
+    # None of the above -> unknown position
+    return UNKNOWN
+
+
+def getCurrentAlignerPositionLabel(current_pos, align):
+    """
+    Determine where lies the current stage position
+    :param current_pos: (dict str->float) Current position of the stage
+    :param align: Sample stage that's being controlled
+    :return: (int) a value representing stage position from the constants LOADING, IMAGING, TILTED, COATING..etc
+    """
+    align_md = align.getMetadata()
```

Comment:
A bit confused. Which component is "align"? Stage or objective aligner? I guess it is the objective aligner. If so, then i think it should not be "sample stage" in the docstring...